src/components/charts-generic/scatter-plot/median.tsx

```diff
@@ -0,0 +1,32 @@
+import { palette } from "src/themes/palette";
+
+import { useChartState } from "../use-chart-state";
+
+import { ScatterPlotState } from "./scatter-plot-state";
+
+export const ScatterPlotMedian = () => {
+  const {
+    medianValue: m,
+    bounds,
+    xScale,
+  } = useChartState() as ScatterPlotState;
+
+  const diamondSize = 8;
+
+  if (!m) {
+    return null;
+  }
+
+  const x = xScale(m);
+  const y = bounds.chartHeight / 2 + bounds.margins.top;
```

Comment:
seems a bit fragile not to rely on a data space value that would be transformed into passed into a yScale.